### ISSUE TITLE
Minify doesn't work anymore with 1.1.0

### ISSUE THREAD

**Issue by jeroencornelissen** [stealjs/steal-tools#591]
I understand there are some change with minify.
But the default behaviour doesn't work anymore.
steal.production.js gets minified but bundles/main.js & bundles/main.css are not minified.

This is our build.js:
```
var stealTools = require("steal-tools");

var buildPromise = stealTools.build({
    main    : "main",
  	config: __dirname + "/package.json!npm"
}, {
  	bundleAssets	: false
});
```

Even with explicitly adding `minify: true` the bundle files are not minified.

**Comment by matthewp** [comment]
cc @m-mujica can you take a look and do some sanity checks?

**Comment by m-mujica** [comment]
I have an idea of what introduced the bug, working on a fix.

**Comment by m-mujica** [comment]
@jeroencornelissen thanks for filing the issue, we just released steal-tools@1.1.1

When you have a chance please try it out and let us know if it works for you.

**Comment by jeroencornelissen** [comment]
@m-mujica , it's working again in 1.1.1!
Thanks a lot!
